doc/startify.txt

```diff
@@ -667,7 +677,7 @@ If you provide a string to it instead, Startify will evaluate it every time
 :Startify is run:
 >
     let g:startify_custom_header =
-          \ 'startify#pad(g:ascii + startify#fortune#boxed())'
+          \ 'startify#pad(s:ascii + startify#fortune#boxed())'
```

Comment:
This will break, because Startify will try to evaluate it outside of the context of the vimrc where `s:ascii` won't exist.